DESCRIPTION

```diff
@@ -30,14 +30,17 @@ Imports:
     mlr3cluster,
     mlr3data,
     mlr3filters,
-    mlr3fselect,
+    mlr3fselect (>= 0.5.1.9000),
```

Comment:
Should we really add version requirements in {mlr3verse}? This makes {mlr3verse} not installable for users which are not on these minimum versions even though other combinations of packages might be fine.

Should such restrictions only live within the respective individual packages?

Reply:
I just add these as a reminder to all of my packages to not merge before a new version is on CRAN.

> This makes {mlr3verse} not installable for users which are not on these minimum versions even though other combinations of packages might be fine

What happens with new functions if someone has the latest mlr3verse version but an old mlr3tuning?

Reply:
Good Q, we should check what happens